src/lib/Form/Type/Role/RoleCopyType.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\EzPlatformAdminUi\Form\Type\Role;
+
+use EzSystems\EzPlatformAdminUi\Form\Data\Role\RoleCopyData;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\SubmitType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class RoleCopyType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add(
+                'new_identifier',
+                TextType::class,
+                ['label' => /** @Desc("Name of a new role") */ 'role_copy.name']
+            )
+            ->add(
+                'save',
```

Comment:
Nitpick: Why not call this action `copy`? It makes more sense since translation says `Copy`.